libopenage/shader/program.cpp

```diff
@@ -1,8 +1,7 @@
-// Copyright 2013-2016 the openage authors. See copying.md for legal info.
+// Copyright 2013-2017 the openage authors. See copying.md for legal info.
 
 #include "program.h"
 
-#include <unistd.h>
```

Comment:
I used that option only as a last-resort thing. The first (and IMO better) option is to identify and eliminate platform-dependent dependencies like this (technically `unistd.h` is part of the POSIX standard, but VS doesn't provide one). Same as above, if the CI is happy, we can remove this, right?